idp/credentials/shib-idp/inc-md-cert-mdq.pem

```diff
@@ -0,0 +1,29 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Just checking with this file since I didn't see a corresponding one deleted in the old config.  This is needed, right?

Reply:
That is for incommon. It may not be needed. I can try without it.

Reply:
Removed.